From cb23f8675e573d2f73745f641daaf682d91fb5b5 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sat, 14 Dec 2019 19:21:29 +0200
Subject: [PATCH 1/1] Update icon to match AOSP

  * from
  https://github.com/phhusson/treble_app/pull/18/commits/730ff460147246e4b1b60980d88c91b326c989b2

	modified:   app/src/main/AndroidManifest.xml
	new file:   app/src/main/res/drawable/ic_settings.xml
---
 app/src/main/AndroidManifest.xml          |  5 ++---
 app/src/main/res/drawable/ic_settings.xml | 10 ++++++++++
 2 files changed, 12 insertions(+), 3 deletions(-)
 create mode 100644 app/src/main/res/drawable/ic_settings.xml

diff --git a/app/src/main/AndroidManifest.xml b/app/src/main/AndroidManifest.xml
index 28b7dad..0e0f28c 100644
--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -9,9 +9,8 @@
     <uses-permission android:name="android.permission.DIAGNOSTIC" />
     <application
         android:allowBackup="true"
-        android:icon="@mipmap/ic_launcher"
+        android:icon="@drawable/ic_settings"
         android:label="@string/app_name"
-        android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true">
 
         <uses-library android:name="android.hidl.manager.V1_0" android:required="false" />
@@ -39,7 +38,7 @@
             <meta-data android:name="com.android.settings.category" android:value="com.android.settings.category.ia.homepage"/>
             <meta-data android:name="com.android.settings.summary" android:resource="@string/desc_activity_top_level"/>
             <meta-data android:name="com.android.settings.title" android:resource="@string/title_activity_top_level"/>
-            <meta-data android:name="com.android.settings.icon" android:resource="@mipmap/ic_launcher_round"/>
+            <meta-data android:name="com.android.settings.icon" android:resource="@drawable/ic_settings"/>
             <meta-data android:name="com.android.settings.order" android:value="-190"/>
         </activity-alias>
 
diff --git a/app/src/main/res/drawable/ic_settings.xml b/app/src/main/res/drawable/ic_settings.xml
new file mode 100644
index 0000000..3c56ed7
--- /dev/null
+++ b/app/src/main/res/drawable/ic_settings.xml
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0"
+        android:tint="?attr/colorControlNormal">
+    <path
+        android:fillColor="@android:color/white"
+        android:pathData="M10.82 12.49c.02-.16.04-.32.04-.49 0-.17-.02-.33-.04-.49l1.08-.82c.1-.07.12-.21.06-.32l-1.03-1.73c-.06-.11-.2-.15-.31-.11l-1.28.5c-.27-.2-.56-.36-.87-.49l-.2-1.33c0-.12-.11-.21-.24-.21H5.98c-.13 0-.24.09-.26.21l-.2 1.32c-.31.12-.6.3-.87.49l-1.28-.5c-.12-.05-.25 0-.31.11l-1.03 1.73c-.06.12-.03.25.07.33l1.08.82c-.02.16-.03.33-.03.49 0 .17.02.33.04.49l-1.09.83c-.1.07-.12.21-.06.32l1.03 1.73c.06.11.2.15.31.11l1.28-.5c.27.2.56.36.87.49l.2 1.32c.01.12.12.21.25.21h2.06c.13 0 .24-.09.25-.21l.2-1.32c.31-.12.6-.3.87-.49l1.28.5c.12.05.25 0 .31-.11l1.03-1.73c.06-.11.04-.24-.06-.32l-1.1-.83zM7 13.75c-.99 0-1.8-.78-1.8-1.75s.81-1.75 1.8-1.75 1.8.78 1.8 1.75S8 13.75 7 13.75zM18 1.01L8 1c-1.1 0-2 .9-2 2v3h2V5h10v14H8v-1H6v3c0 1.1.9 2 2 2h10c1.1 0 2-.9 2-2V3c0-1.1-.9-1.99-2-1.99z"/>
+</vector>
-- 
2.17.1

